src/relay/relay.py

```diff
@@ -103,20 +102,10 @@ def relay_status(context: click.Context) -> HTTPResponse:
 
     Returns:
         An HTTP response with a status of OK and a JSON payload containing
-        status information (currently, the output from `ls` listing the files
-        in the upload directory).
+        status information.
     """
     logging.info("request to report status")
     body = {"disk utilization": get_disk_utilization_str(context.meta[CTX_DIRECTORY])}
```

Comment:
While you're here, it might be more useful to return a real JSON object rather than just a "utilization string": e.g.,

```json
{
    "disk_utilization": {
        "bytes_used": 1200000,
        "bytes_remaining": 80000000000,
        "string": "0.00001% full, 80Gb remaining"
    }
}
```

(PS: don't criticize the numbers: I just typed random 0s 😆 )

Reply:
Dang...I just got the unit tests working!...but, OK....  🥴